internal/traces/traces.go

```diff
@@ -25,7 +25,15 @@ func WithTraceID(ctx context.Context) context.Context {
 
 func Logger(ctx context.Context) *logrus.Entry {
 	return logrus.WithField("trace.id", TraceID(ctx))
+}
 
+// Creates a new instance of the logger which only logs WARNING events.
+// We use it as logger for the GRPC events which are hardcoded as INFO level and pollute the logs.
+func WarnLogger(ctx context.Context) *logrus.Entry {
+	warnLevel, _ := logrus.ParseLevel("warn")
+	warnLogger := logrus.New();
+	warnLogger.SetLevel(warnLevel)
```

Comment:
I think we can use `logrus.WarnLevel` directly, instead of parsing the string to it.